runtime/gc_vlhgc/GlobalMarkingScheme.cpp

```diff
@@ -1128,6 +1128,35 @@ MM_GlobalMarkingScheme::scanContinuationObjects(MM_EnvironmentVLHGC *env)
 	env->getGCEnvironment()->_continuationObjectBuffer->flush(env);
 }
 
+void
+MM_GlobalMarkingScheme::iterateContinuationObjects(MM_EnvironmentVLHGC *env)
+{
+	J9JavaVM *vm = (J9JavaVM*)env->getOmrVM()->_language_vm;
+	J9JITConfig *jitConfig = vm->jitConfig;
+	if (NULL == jitConfig->methodsToDelete) {
```

Comment:
can we generalize this with a check like 'anything registered to J9HOOK_MM_WALKCONTINUATION' ?

Reply:
move the check for `jitConfig->methodsToDelete` to RootScanner